unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityInformation/SmartItems/Parameters/Actions/SmartItemActionEventAdapter.cs

```diff
@@ -36,9 +35,9 @@ public void SetContent(ActionEvent actionEvent)
 
     void SelectedEntity(int number)
     { 
-        SmartItemComponent component = (SmartItemComponent)filteredList[number].rootEntity.GetBaseComponent(CLASS_ID_COMPONENT.SMART_ITEM);
-        selectedComponent = component;
-        GenerateActionDropdownContent(component.model.actions);
+        filteredList[number].rootEntity.TryGetBaseComponent(CLASS_ID_COMPONENT.SMART_ITEM, out BaseComponent component);
```

Comment:
We should check if we found the component here, it's unlikely, but better be safe than sorry.